samples/testsuite-performance/config/project.properties

```diff
@@ -82,6 +82,7 @@ com.xceptance.xlt.loadtests.TLWSimpleURL.class = com.xceptance.xlt.performance.t
 #   contentTypePattern [c] ... Reg-ex defining a matching response content type
 #   statusCodePattern [s] .... Reg-ex defining a matching status code
 #   urlPattern [u] ........... Reg-ex defining a matching request URL
+#   httpMethodPattern [m] .... Reg-ex defining a matching HTTP method
```

Comment:
Do we want to shorten this property to `methodPattern` instead (so that users can link it with the appropriate category key `m` more easily)? IMO the prefix 'http' is not really necessary in this context (similar to `statusCodePattern`).

Reply:
Yup, done.